wp-modules/editor/tests/UtilsTest.php

```diff
@@ -413,4 +413,14 @@ public function test_has_pattern_block( $content, $expected ) {
 			has_pattern_block( $content )
 		);
 	}
+
+	/**
+	 * Tests prepend_textdomain.
+	 */
+	public function test_prepend_textdomain() {
+		$this->assertStringEndsWith(
+			'my-new-pattern',
+			prepend_textdomain( 'my-new-pattern' )
```

Comment:
Not a complete test, but it's hard to mock `wp_get_theme()` in `prepend_textdomain()`, to mock the textdomain being empty.